cmd/flexvolume/cmd/init.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2017 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/rook/rook/pkg/util/sys"
+	"github.com/spf13/cobra"
+)
+
+var (
+	initCmd = &cobra.Command{
+		Use:   "init",
+		Short: "Initialize the volume plugin",
+		RunE:  initPlugin,
+	}
+)
+
+func init() {
+	RootCmd.AddCommand(initCmd)
+}
+
+func initPlugin(cmd *cobra.Command, args []string) error {
+
+	moduleName := "rbd"
```

Comment:
I would make a const out of `moduleName` even if it is just used here.

Reply:
sounds good. Will do.